conftest.py

```diff
@@ -1,29 +1,36 @@
+"""PyTest Fixtures."""
```

Comment:
I don't think conftest should be moved.

Reply:
Again, that is not up to personal preferences. This file must be at the root of all tests, now that we have tests in test/ and also in lib/ansiblelib/rules/ we must put it at repository root.

Reply:
Are you sure you cannot set its path as a pytest invocation arg?

Reply:
Besides, pytest traverses the dir layout and will probably pick it up anyway